.github/workflows/image-push-release.yml

```diff
@@ -109,7 +109,8 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: ${{ env.IMAGE_NAME }}
-          tag-latest: false
```

Comment:
didnt find this field in metadata-action v4 docs  so i used whats documented.

https://github.com/docker/metadata-action